### PR TITLE
Remove mention of piracy tools in help text

### PR DESCRIPTION
**Pull request by felix920506**
**Changes**

Change the labels for the App Name in API keys section to remove mentions of piracy tools per @mcarlton00 's suggestion on Matrix

image

**Issues**

None

### CODE REVIEW

File: src/strings/en-us.json [hunk 1]
```diff
@@ -543,8 +543,8 @@
     "LabelAllowedRemoteAddresses": "Remote IP address filter",
     "LabelAllowedRemoteAddressesMode": "Remote IP address filter mode",
     "LabelAllowHWTranscoding": "Allow hardware transcoding",
-    "LabelAppName": "App name",
-    "LabelAppNameExample": "Example: Sickbeard, Sonarr",
+    "LabelAppName": "Human Readable Name",
+    "LabelAppNameExample": "A human readable name for identifying API keys. This setting will not affect functionality",
```
Comment (thornbill) [inline src/strings/en-us.json:547]:
```suggestion
    "LabelAppNameExample": "A human readable name for identifying API keys. This setting will not affect functionality.",
```

File: src/strings/en-us.json [hunk 2]
```diff
@@ -543,8 +543,8 @@
     "LabelAllowedRemoteAddresses": "Remote IP address filter",
     "LabelAllowedRemoteAddressesMode": "Remote IP address filter mode",
     "LabelAllowHWTranscoding": "Allow hardware transcoding",
-    "LabelAppName": "App name",
-    "LabelAppNameExample": "Example: Sickbeard, Sonarr",
+    "LabelAppName": "Human Readable Name",
```
Comment (thornbill) [inline src/strings/en-us.json:546]: I think leaving this as "App name" would be fine with the improved example/help text. 🤷‍♂️

Comment (felix920506) [reply src/strings/en-us.json:546]: Maybe should also be fine

Comment (thornbill) [inline src/strings/en-us.json:546]:
```suggestion
    "LabelAppName": "App name",
```

**Review by thornbill — APPROVED** [review]

**Pull request merged**